internal/healthchecks/ports_check.go

```diff
@@ -45,6 +45,18 @@ func checkIfPortAvailable(host string, port string, network string) (bool, error
 }
 
 func (c PortsCheck) RunCheck(logger *log.Logger) error {
+	fbActive, err := isSubagentActive("google-cloud-ops-agent-fluent-bit")
+	if err != nil {
+		return err
+	}
+	ocActive, err := isSubagentActive("google-cloud-ops-agent-opentelemetry-collector")
```

Comment:
I think the name of the `opentelemetry-collector` subagent in windows is different to the one in linux. See https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/ops_agent_test.go#L144-L150

Reply:
That's the process name -- we're checking the service name here, which looks correct: https://github.com/GoogleCloudPlatform/ops-agent/blob/master/cmd/ops_agent_windows/main_windows.go#L120

Reply:
Ok! Thanks for the clarification.